assignments.js

```diff
@@ -5,10 +5,11 @@ if (randomNumber === 1) {
 } else {
     const anotherNumber = randomNumber === 5;
     console.log(`Now we have a number ${anotherNumber}`)
+
 } if (randomNumber > 8) {
     console.log(`This number is somewhere in the middle, number ${randomNumber}`);
```

Comment:
You should put `if` statement in new line after braces closed, like this

```suggestion
}

if (randomNumber > 8) {
    console.log(`This number is somewhere in the middle, number ${randomNumber}`);
```